pulsar/consumer_partition.go

```diff
@@ -733,6 +734,7 @@ func (pc *partitionConsumer) internalAck(req *ackRequest) {
 
 func (pc *partitionConsumer) MessageReceived(response *pb.CommandMessage, headersAndPayload internal.Buffer) error {
 	pbMsgID := response.GetMessageId()
+	ackSet := response.GetAckSet()
```

Comment:
It seems that ackSet not handled to judge which msg is acked.
It maybe cause duplicated consuming.
Would you please have a look, thanks a lot.

Reply:
The `ackSet` should be parsed into a `BitSet` to check if the messages should be skipped when constructing the messages from a batch.